app/models/katello/host/subscription_facet.rb

```diff
@@ -106,6 +106,10 @@ def self.find_host(name, organization)
         hosts.first
       end
 
+      def self.sanitized_name(name)
```

Comment:
Kind of a nitpick, but I'd expect this to be a verb since its expecting a parameter and then returning a changed value.